File: 3st/Jaeyoung/A 강조하기.js
function solution(myString) {
    var answer = '';
    answer = myString.split('')
    for( var i = 0 ; i < answer.length ; i++ ) {
        if( answer[i] === "a" || answer[i] === "A" ) answer[i] = answer[i].toUpperCase();
        else answer[i] = answer[i].toLowerCase();
    }
    return answer.join('');
}0

File: 1st/Jaehee/A 강조하기.java
class Solution {
    public String solution(String myString) {

        StringBuilder answer = new StringBuilder();
        
        for (int i = 0; i < myString.length(); i++) {
            char ch = myString.charAt(i);
            
            if (ch == 'a') {

                answer.append('A');
            } else if (Character.isUpperCase(ch) && ch != 'A') {
                answer.append(Character.toLowerCase(ch));
            } else {

                answer.append(ch);
            }
        }
        
        return answer.toString();
    }
}